x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/XPackFeatures.java

```diff
@@ -20,6 +20,7 @@
 public class XPackFeatures implements FeatureSpecification {
     public static final NodeFeature LOGSDB_TELEMETRY = new NodeFeature("logsdb_telemetry");
     public static final NodeFeature LOGSDB_TELMETRY_STATS = new NodeFeature("logsdb_telemetry_stats");
+    public static final NodeFeature SPARSE_VECTOR_STORE_SUPPORT = new NodeFeature("mapper.sparse_vector.store_support");
```

Comment:
Why not to put this feature into the `MapperFeatures` class?

Reply:
Moved to MapperFeatures